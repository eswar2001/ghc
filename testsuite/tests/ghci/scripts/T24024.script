:set -fdefer-type-errors
:kind Int Int
