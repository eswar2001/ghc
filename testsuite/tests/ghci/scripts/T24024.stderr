
<interactive>:1:1: error: [GHC-83865]
    • Expected kind ‘* -> k0’, but ‘Int’ has kind ‘*’
    • In the type ‘Int Int’
